common/config/subspaces/default/common-versions.json

```diff
@@ -48,7 +44,18 @@
    * will recalculate all version selections.
    */
   // "implicitlyPreferredVersions": false,
-
+  /**
+   * If you would like the version specifiers for your dependencies to be consistent, then
+   * uncomment this line. This is effectively similar to running "rush check" before any
+   * of the following commands:
+   *
+   *   rush install, rush update, rush link, rush version, rush publish
+   *
+   * In some cases you may want this turned on, but need to allow certain packages to use a different
+   * version. In those cases, you will need to add an entry to the "allowedAlternativeVersions"
+   * section of the common-versions.json.
+   */
+  "ensureConsistentVersions": true,
```

Comment:
This won't pass schema validation until Rush is published with this change and bumped in this repo.

Reply:
Bumped rush version in this MR